frontend/src/features/properties/map/MapView.tsx

```diff
@@ -76,6 +78,10 @@ const MapView: React.FC<MapViewProps> = (props: MapViewProps) => {
   };
 
   const { showSideBar } = useParamSideBar();
+
+  const location = useLocation();
+  const urlParsed = queryString.parse(location.search);
+  const disableFilter = urlParsed.sidebar === 'true' ? true : false;
```

Comment:
Now the search filter will be disabled when the side bar is open, instead of being squished to the side